.travis.yml

```diff
@@ -8,7 +8,7 @@ python:
 cache: pip
 install:
   - pip install -U pip wheel
-  - pip install tensorflow==1.0.1
+  - pip install tensorflow==1.3.0rc0
```

Comment:
can we later specify this without a version number so we just always test against the latest stable? Which is the goal anyways, or rather work with the latest stable GPflow which hopefully follows the latest stable TF closely.